src/traces/sunburst/helpers.js

```diff
@@ -52,10 +52,11 @@ exports.getPtId = function(pt) {
 };
 
 exports.getPtLabel = function(pt) {
-    return pt.data.data.label;
+    var label = pt.data.data.label;
+    return label === undefined ? pt.data.data.pid : label;
```

Comment:
Hmm. I don't like this. This helper is supposed to return the `label` not the "label or the pid".

I'd would leave `getPtLabel` along, and move this extra logic in `getParent` or make another helper.

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/4219/commits/df5e1219dd2892b29d9c6a2fe5ec9f3dd3e4f2ae.

Reply:
Thanks!